tmk_core/common/keyboard.c

```diff
@@ -64,20 +64,41 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 #ifdef MATRIX_HAS_GHOST
+static uint16_t matrix_ghost_check[MATRIX_ROWS];
```

Comment:
This could probably be built dynamically to save a bit of memory, perhaps at the cost of slightly bigger flash memory usage. The AVR devices don't have much memory, so I think it's worth trying to save as much as possible.